### PR TITLE
Update power-plant.yaml

### PR DESCRIPTION
**Pull request by sandrinecharousset**
Includes following corrections:
- traditionnal => traditional (for some instances)
- reservoir => Reservoir (for some instances)
- some run of river variables were wrongly named reservoir
- added missing variable turbine efficiency

### CODE REVIEW

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 1]
```diff
@@ -403,6 +435,11 @@ Rate Frequency Containment Reserve|Electricity|Gas|OCGT:
    description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
                 for a typical Open Cycle Gas Turbine power plant
    unit: "%"
+   
+Rate Frequency Containment Reserve|Electricity|Gas|CCGT:
+   description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
+                for a typical Open Cycle Gas Turbine power plant
```
Comment (erikfilias) [inline nomenclature/definitions/variable/technology/power-plant.yaml:441]: ...for a typical Combined Cycle Gas Turbine power plant.

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 2]
```diff
@@ -437,6 +474,11 @@ Rate Automatic Frequency Restoration Reserve|Electricity|Gas|OCGT:
                 for a typical Open Cycle Gas Turbine power plant
    unit: "%"
 
+Rate Automatic Frequency Restoration Reserve|Electricity|Gas|CCGT:
+   description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
+                for a typical Open Cycle Gas Turbine power plant
```
Comment (erikfilias) [inline nomenclature/definitions/variable/technology/power-plant.yaml:479]: ...for a typical Combined Cycle Gas Turbine power plant.

**Review by erikfilias — COMMENTED** [review]
Looks good!

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 1 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/technology/power-plant.yaml:441]: thanks

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 2 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/technology/power-plant.yaml:479]: :-)

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 3]
```diff
@@ -571,7 +613,7 @@ Number of Units|Electricity|Hydro|Pumped Storage:
    description: Number of units of kind "Pumped Storage "
    unit:
 
-Maximum Active Power|Electricity|Hydro|Pumped Storage:
+Maximum Active power|Electricity|Hydro|Pumped Storage:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/technology/power-plant.yaml:616]: For consistency...
```suggestion
Maximum Active Power|Electricity|Hydro|Pumped Storage:
```

**Review by danielhuppmann — APPROVED** [review]
Thanks @sandrinecharousset - I took the liberty of fixing one minor issue directly (rewriting `power` to `Power` in a variable name.

**Pull request merged**